playwright/src/fixtures/content.page.ts

```diff
@@ -285,6 +325,7 @@ class ContentPage {
       await this.page.mouse.down()
       await this.page.mouse.move(boundary.width - 20 + boundary.x, boundary.y + boundary.height - 10)
       await this.page.mouse.up()
+      
```

Comment:
@gregfitch this function `selectText` which I use to select text for highlighting sometimes drags beyond the range of the selector and selects extra text. (I have used only paragraph selectors, so at any point it should be selecting just one paragraph but occassionally it drags into the next para also).
Is there any way this can be prevented

Reply:
This is generally due to the page layout, especially if the last line of text is less than half the width of the content block. I'm noticed browsers sometimes move to the next text block if you run beyond the closing tags 'location'. There's not much you can do to combat this as the block size changes with screen size (or browser), other than try to move fewer extra pixels during the mouse drag. This is one of the reasons why I used random text selects instead of full paragraphs most of the time, especially if it could grab parts that aren't in the highlightable bounds.

Reply:
ok, thank you.